src/signin.js

```diff
@@ -220,72 +226,75 @@ function logInLoadPreferences (context) {
         }
-        context.preferencesFile = preferencesFile
-        return resolve(context)
-      },
-      function (err) { // Really important to look at why
-        let status = err.status
-        let message = err.message
-        console.log('HTTP status ' + status + ' for pref file ' + preferencesFile)
-        let m2
-        if (status === 401) {
-          m2 = 'Strange - you are not authenticated (properly logged on) to read preferences file.'
-          alert(m2)
-        } else if (status === 403) {
-          if (differentOrigin()) {
-            m2 = 'Unauthorized: Assuming prefs file blocked for origin ' + window.location.origin
-            context.preferencesFileError = m2
+
+        // //// Load preferences file
+        return kb.fetcher
+          .load(preferencesFile, {withCredentials: true})
+          .then(function () {
+            if (progressDisplay) {
+              progressDisplay.parentNode.removeChild(progressDisplay)
+            }
+            context.preferencesFile = preferencesFile
             return resolve(context)
-          }
-          m2 = 'You are not authot=rized to read your prefernces file. This may be because you are using an trusted web app.'
-          console.warn(m2)
-        } else if (status === 404) {
-          if (confirm('You do not currently have a Preferences file. Ok for me to create an empty one? ' + preferencesFile)) {
-            // @@@ code me  ... weird to have a name o fthe file but no file
-            alert('Sorry I am not prepared to do this ... please  create an empty file at ' + preferencesFile)
-            return complain(new Error('Sorry No code yet to craete a preferences fille at '))
-          } else {
-            reject(new Error('User declined to craete a preferences fille at ' + preferencesFile))
-          }
-        } else {
-          m2 = 'Strange: Error ' + status + ' trying to read your preferences file.' + message
-          alert(m2)
-        }
-      }) // load prefs file then
-    }, err => { // Fail initial login load prefs
-      reject(new Error('(via loadPrefs) ' + err))
-    }, err => reject(err))
+          })
+          .catch(function (err) { // Really important to look at why
+            let status = err.status
+            let message = err.message
+            console.log('HTTP status ' + status + ' for pref file ' + preferencesFile)
+            let m2
+            if (status === 401) {
+              m2 = 'Strange - you are not authenticated (properly logged on) to read preferences file.'
+              alert(m2)
+            } else if (status === 403) {
+              if (differentOrigin()) {
+                m2 = 'Unauthorized: Assuming prefs file blocked for origin ' + window.location.origin
+                context.preferencesFileError = m2
+                return resolve(context)
+              }
+              m2 = 'You are not authothorized to read your preferences file. This may be because you are using an untrusted web app.'
```

Comment:
```suggestion
              m2 = 'You are not authorized to read your preferences file. This may be because you are using an untrusted web app.'
```
Sorry; a typo replaced a typo :-/